src/react-apps/applications/ux-editor/src/containers/FormFiller.tsx

```diff
@@ -42,8 +60,10 @@ export class FormFillerComponent extends React.Component<IFormFillerProps, IForm
   public saveFormData = () => {
     const altinnWindow: IAltinnWindow = window as IAltinnWindow;
     const { reportee, org, service, instanceId } = altinnWindow;
-    FormFillerActionDispatchers.submitFormData(`
+    if (window.location.pathname.split('/')[1].toLowerCase() === 'runtime') {
```

Comment:
Another check for runtime that is not needed?

Reply:
Yes, removed.